platform/macos/godot_application.mm

```diff
@@ -63,7 +63,7 @@ - (void)activateApplication {
 	NSString *nsbundleid_env = [NSString stringWithUTF8String:(bundled_id != nullptr) ? bundled_id : ""];
 	NSString *nsbundleid = [[NSBundle mainBundle] bundleIdentifier];
 	if (nsappname == nil || isatty(STDOUT_FILENO) || isatty(STDIN_FILENO) || isatty(STDERR_FILENO) || ![nsbundleid isEqualToString:nsbundleid_env]) {
-#if DEV_ENABLED
+#ifdef TOOLS_ENABLED
 		if (!OS_MacOS::is_debugger_attached())
```

Comment:
It's a bit asking for trouble that this doesn't use brackets. If we add more code below it will break the logic in unexpected ways.

Reply:
Replaced it with (same pattern is used in other similar cases):
```
#else
		{
#endif
```